triggers/sg_jira_event_trigger.py

```diff
@@ -9,6 +9,16 @@
 import logging
 import requests
 import urlparse
+
+try:
+    # Allow users to define their sensible data in a .env file and
+    # load it in environment variables with python-dotenv.
+    # https://pypi.org/project/python-dotenv/
+    from dotenv import load_dotenv
+    load_dotenv(override=True)
+except ImportError:
```

Comment:
This try/except is probably not required anymore, right, since dotenv is installed?